hare.spec

```diff
@@ -47,7 +47,7 @@ BuildRequires: python36-pip
 BuildRequires: python36-setuptools
 
 Requires: consul >= 1.7.0, consul < 1.10.0
-Requires: facter >= 3.14.8
+Requires: puppet-agent >= 6.13.0
```

Comment:
I'm afraid this won't fix the issue, since `puppet-agent` is Seagate's custom rpm which is not available for external users.

Reply:
@andriytk, I can find puppet-agent rpms at https://yum.puppetlabs.com/puppet/el/7/x86_64/ also at (https://puppet.com/docs/puppet/7/install_agents.html#install_agents)  , I think within Seagate it is installed from Seagate's private repo but if we can document the appropriate external repo to be configured for the community then I think it should be fine, no? cc @vaibhavparatwar, @knekrasov

Reply:
Nice, didn't know that. Do they provide the needed version of facter?